docs/sidebars.js

```diff
@@ -42,6 +42,7 @@ module.exports = {
         label: "Tabs",
         items: [
           "tabs",
+          "vertical-tabs-w-line",
```

Comment:
```suggestion
          "line-tab",
```
i guess we wont be having line-tab horizontal 😄